data.js

```diff
@@ -208,6 +208,37 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- literature wrote \n- Press mentions",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Their outcome is measured well. These are some of (2018)[https://www.wri.org/annualreport/2018-19#page=4] and their are (more)[https://www.wri.org/our-work/outcomes].",
```

Comment:
wrong brackets. Also, for future reference: the format we're using is called Markdown. It's very common in software engineering (but sadly only there -.-):
https://duckduckgo.com/?q=markdown+cheat+sheet

Reply:
@yeah800 tip for future PRs (pull requests): Read it through yourself for common & easy mistakes to reduce back-and-forth

Reply:
fixed the link